Documentation/Design/UserStory_WorldInteraction.htm

```diff
@@ -0,0 +1,86 @@
+<html>
+<head></head>
```

Comment:
That said, raising SpecFlow is interesting.  Maybe we can do something with that.  Maybe we track an issue for exploration there?